tests/main.js

```diff
@@ -1976,3 +1976,14 @@ test('Creates a manifest file with the list of created functions if the `manifes
     t.is(fn.path, file.path)
   })
 })
+
+testBundlers('Correctly follows node_modules via symlink', [ESBUILD, ESBUILD_ZISI], async (bundler, t) => {
+  const { tmpDir } = await zipNode(t, 'node-module-symlinks', {
+    opts: { config: { '*': { nodeBundler: bundler } } },
+  })
+
+  // eslint-disable-next-line import/no-dynamic-require, node/global-require
```

Comment:
We'll need to remove these once we land https://github.com/netlify/zip-it-and-ship-it/pull/682.

Reply:
#682 is only about fixtures, not about test code. We could add `// eslint-disable-file import/no-dynamic-require, node/global-require` to the top of this file instead of line-by-line, though!